lib/packwerk.rb

```diff
@@ -7,6 +7,9 @@
 
 # Provides String#pluralize
 require "active_support/core_ext/string"
+require 'packwerk/diagnostics'
+Packwerk::Diagnostics.log('Requiring railtie if Rails is defined', __FILE__)
+require 'packwerk/railtie' if defined?(Rails)
```

Comment:
With this pattern, packwerk will add its rake task if run via `bin/rake`, but otherwise won't interact with rails if rails isn't already loaded.